sup3r/bias/bias_correct_means.py

```diff
@@ -0,0 +1,949 @@
+"""Classes to compute means from vortex and era data and compute bias
+correction factors.
+"""
+
+
+import calendar
+import json
+import logging
+import os
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import h5py
+import numpy as np
+import pandas as pd
+import rioxarray
+import xarray as xr
+from rex import Resource
+from scipy.interpolate import interp1d
+from sklearn.neighbors import BallTree
+
+from sup3r.postprocessing.file_handling import OutputHandler, RexOutputs
+from sup3r.preprocessing.feature_handling import Feature
+from sup3r.utilities import VERSION_RECORD
+
+logger = logging.getLogger(__name__)
+
+
+class VortexMeanPrepper:
+    """Class for converting monthly vortex tif files for each height to a
+    single h5 files containing all monthly means for all requested output
+    heights.
+    """
+
+    def __init__(self, path_pattern, in_heights, out_heights, overwrite=False):
+        """Parameters
```

Comment:
Minor but i wonder if you need a line break to get the parameters to show up in the docs page